src/main/java/at/tugraz/sw20asd/lang/ui/Layout.java

```diff
@@ -9,11 +9,14 @@ public Layout(){
         this.setDisable(false);
 
         Tab addTab = new Tab("Add vocabulary");
+        Tab addTab2 = new Tab("Overview");
```

Comment:
Please, use more descriptive names. The `addTab` is named as such, as it is used for adding a new vocabulary. `addTab2` does not convey any intended functionality.